src/Core/src/Handlers/HybridWebView/HybridWebViewHandler.cs

```diff
@@ -18,11 +18,33 @@
 using System.Threading.Tasks;
 using Microsoft.Maui.Storage;
 using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Linq;
+using System.Text.Json;
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class HybridWebViewHandler : IHybridWebViewHandler
+	internal interface IHybridWebViewTaskManager
```

Comment:
A note for the person that wants to make this public, we may have to make this a bit different and instead use an object instance in the handler and/or make the API not expose the dictionary but rather a more basic Get/Add/Remove method set to avoid leaking implementation details.

Reply:
No matter what, it's not designed to 'just be made public'. I think that's just something to decide in the future depending on what direction to go in. I wanted to avoid doing a lot of design work and code for something that would likely change a lot in the future. The interface for now is merely to allow separation of the logic in terms of how it's invoked.